tests/integ/test_large_query_breakdown.py

```diff
@@ -158,6 +158,64 @@ def test_save_as_table(session, large_query_df):
     assert history.queries[3].sql_text.startswith("DROP  TABLE  If  EXISTS")
 
 
+def test_in_with_subquery_multiple_query(session):
+    if not session.sql_simplifier_enabled:
```

Comment:
@sfc-gh-aalam i think we can separate the test with fix to two different pr here